src/i18n.ts

```diff
@@ -111,19 +103,19 @@ export class I18n<C extends Context = Context> {
    * Gets a message by its key from the specified locale.
    * Alias of `translate` method.
```

Comment:
Unrelated but would be cool

```suggestion
   * Alias of `translate`.
```